rising/transforms/functional/affine.py

```diff
@@ -120,8 +116,7 @@ def affine_image_transform(image_batch: torch.Tensor,
         missing_dims = len(image_batch.shape) - len(image_size)
         new_size = (*image_batch.shape[:missing_dims], *new_size)
 
-    matrix_batch = matrix_batch.to(device=image_batch.device,
-                                   dtype=image_batch.dtype)
+    matrix_batch = matrix_batch.to(image_batch)
```

Comment:
From which PyTorch version on is this supported? I want to avoid having some implicit constraints here

Reply:
Supported since the beginning / since `to` is supported (0.4.0)